lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -58,6 +58,9 @@ class UnusedBytes extends Audit {
   }
 
   /**
+   * Estimates the number of bytes this network record consumed on the network based on the
+   * uncompressed size, uses the actual transfer size from the network record if applicable.
```

Comment:
maybe something like `Estimates the number of bytes this network record would have consumed on the network based on the uncompressed size totalBytes, ...`? What `totalBytes` is is probably the thing that needs the most explanation

Reply:
done